RELEASE.md

```diff
@@ -0,0 +1,3 @@
+Release type: patch
+
+Fix builtin datetime, date and time classes conversion
```

Comment:
```suggestion
Improve datetime, date and time types conversion. Removes aniso dependency
and also adds support for python types, so now we can do use python datetime's types 
instead of our custom scalar types.
```